src/Styles/ContactPage.css

```diff
@@ -1,32 +1,103 @@
-.contactUs h1,h2{
+body{
+    background-image: url(./../../public/C000BG01.svg);
+    background-size: cover;
+}
+
+.contactWordLocation{
+    float: left;
+    width: 55%;
+    padding-left: 120px;
+    padding-top: 80px;
+    padding-right: 0px;
+}
+
+.contactWord h1{
+    font-weight: bold;
     width: 100%;
-    text-align: center;
+    font-family: Kristen ITC;
+    font-size: 200xp;
+    color: #ffffff;
 }
 
-.contactUs h1{
-    color: #71518A;
+.contactWord h2{
+    width: 100%;
+    font-family: Kristen ITC;
+    font-size: 200xp;
```

Comment:
For **all the numerical settings**, change `px` into `vw` or `vh` for responsive looks so that the sizes and locations of things will adjust according to the screen size. For example, right now when I have a smaller window, the texts and background are not in place:
![Screen Shot 2022-06-18 at 5 40 41 PM](https://user-images.githubusercontent.com/49298813/174457806-4968b23a-9d31-465b-b059-d662c9674f17.png)